src/data/selectors/grades.js

```diff
@@ -1,3 +1,40 @@
-const getBulkManagementHistory = state => state.grades.bulkManagement.history;
+import { formatDateForDisplay } from '../../data/actions/utils';
+
+const getRowsProcessed = (data) => {
+  const {
+    processed_rows: processed,
+    saved_rows: saved,
+    total_rows: total,
+  } = data;
+  return {
+    total,
+    successfullyProcessed: saved,
+    failed: processed - saved,
+    skipped: total - processed,
+  };
+};
+
+const transformHistoryEntry = (historyRow) => {
+  const {
+    modified,
+    original_filename: originalFilename,
+    data,
+    ...rest
+  } = historyRow;
+
+  const timeUploaded = formatDateForDisplay(new Date(modified));
```

Comment:
The wireframes include a timezone, which I think is a useful piece of information for course team members to orient themselves. Is that easy to add?

Reply:
We should probably apply it across the board if we think it's useful.

My understanding is that this timezone is pulled out of the browser, and that it's unlikely to reflect the user's preference as configured elsewhere in the platform. That's why I'm a little leery of opening that can of worms for right now.

Reply:
by royal edict, datetimess are to be shown in UTC. I also added them to the other place in this interface they're shown as such